learnEcl/2100-topn.md

```diff
@@ -5,7 +5,7 @@ slug: topn
 
 # TOPN
 
-In a sorted dataset, TOPN returns the n number of records from the row 1 of the dataset.
+In a sorted dataset, TOPN returns the first n number of records in the specified sort order from the dataset.
```

Comment:
The dataset does not have to be sorted.  That is one of the performance perks of using TOPN over SORT.